### PR TITLE
Update FAO data for orchard/crop quest to 2019/2020 one

### PR DESCRIPTION
**Pull request by rugk**
See https://github.com/rugk/crops-parser/pull/59 for what has changed in the data generation.
As you can see from the diff, apparently, we also got better coverage of more countries and a bigger crop list for smaller countries.

Basically, it's again just the most harvested crops (by area) from 2019/2020.

Also [the here removed duplicate YAML lines should be removed with the new fixed data parsing](https://github.com/streetcomplete/StreetComplete/commit/6572b98f6c7734c4e8c1b6f595d6171cfa6fd867#diff-5ca4690e037bfca49b7b5989b6bfd691970c98bc9718e3eca107268841deb118R14) – or, to be more exact, with the use of FAOs own DB for country names and proper attribution (e.g. China is now not duplicated as "China, mainland" is now correctly used for `CN` - see the upstream repository for more information.) 

## Canary Island edit included 

Also manually [ported the Canary Island edit again](https://github.com/streetcomplete/StreetComplete/commit/cebdc28c118183de7fc8203ee5e72ac0bccdd7e8), cf. PR https://github.com/streetcomplete/StreetComplete/pull/2596.

### CODE REVIEW

File: res/country_metadata/orchardProduces.yml [hunk 1]
```diff
@@ -1,214 +1,205 @@
...
+DO: [cacao, coffee, coconut, avocado, banana, lemon, lime, orange, papaya, cashew_nut, pineapple, chilli_pepper, sweet_pepper, tomato, guava, mango, mangosteen, grapefruit, sisal]
+DZ: [olive, date, grape, fig, almond, apple, apricot, tomato, chilli_pepper, sweet_pepper, pear, peach, orange, plum, lemon, lime, cherry]
+EC: [cacao, banana, coffee, orange, guava, mango, mangosteen, rubber, lemon, lime, avocado, pineapple, papaya, agave, pear, coconut, chilli_pepper, sweet_pepper, pepper]
+EE: [strawberry, apple, raspberry, plum, tomato]
+EG: [tomato, guava, mango, mangosteen, orange, olive, grape, date, banana, apple, fig, chilli_pepper, sweet_pepper, peach, strawberry, lemon, lime, apricot, walnut]
+ES: [olive, grape, almond, orange, peach, tomato, lemon, lime, chestnut, apple, chilli_pepper, sweet_pepper, pear, apricot, fig, avocado, plum]
+# Canary Islands has very different climates, biomes and culture (manually edited)
+IC: [olive, grape, almond, orange, peach, tomato, lemon, lime, chestnut, apple, chilli_pepper, sweet_pepper, pear, apricot, fig, avocado, plum, pineapple, mango, avocado]
```
Comment (rugk) [inline res/country_metadata/orchardProduces.yml:60]: BTW @erik55 note as I only saw your https://github.com/streetcomplete/StreetComplete/pull/2596 now, note you may also list your added items at the beginning if you think they are more often grown there in contrast to the other ones.

One could e.g. reorder it like this (of course I have no idea whether that is somewhat realistic or so, it's just a guess):

```suggestion
# Canary Islands has very different climates, biomes and culture (manually edited)
IC: [fig, avocado, plum, pineapple, mango, avocado, apricot, olive, grape, almond, orange, peach, tomato, lemon, lime, chestnut, apple, chilli_pepper, sweet_pepper, pear]
```

For now, I just re-added the ones you added at the end, as you did, to keep the current way as it is.

**Comment by rugk** [comment]
And yes, [apparently Strawberries are a popular thing in Norway](https://www.youtube.com/watch?v=jFdyO8Zr8EQ)… :astonished: :upside_down_face:

**Comment by westnordost** [comment]
Looks good to me, thanks! Are you done with this PR? I noticed some more commits you did since you opened it.

**Comment by rugk** [comment]
No I am done. I just wanted to do that Canary Island thing in a new commit in case we want to revert it/not include it or whatever…

**Pull request merged**